cypress/support/index.js

```diff
@@ -14,7 +14,4 @@ import 'cypress-plugin-retries';
 // Add login cookies to whitelist to preserve it
 beforeEach(() => {
     Cypress.Cookies.preserveOnce('MMAUTHTOKEN', 'MMUSERID', 'MMCSRF');
-
-    cy.apiUpdateConfig({TeamSettings: {MaxUsersPerTeam: 500}});
```

Comment:
This is causing to conflict whenever the spec file is logging in as another user since `cy.apiUpdateConfig` logged in as sysadmin.